packages/core/testing/src/test_bed.ts

```diff
@@ -726,6 +735,10 @@ export class TestBedImpl implements TestBed {
         TEARDOWN_TESTING_MODULE_ON_DESTROY_DEFAULT;
   }
 
+  shouldBeManualDeferBlockBehavior(): boolean {
+    return this._instanceDeferBlockBehavior === DeferBlockBehavior.Manual;
+  }
```

Comment:
nit: we can also just return a current value and assert whether we have a correct one in tests:
```suggestion
  getDeferBlockBehavior(): boolean {
    return this._instanceDeferBlockBehavior;
  }
```

Reply:
I wrote it this way to be consistent with all the methods before it, which follow the `shouldX` pattern. This is only used for test bed unit tests.